### PR TITLE
[refine](Operator) When _stop_emplace_flag is not set to true, perform batch processing on the block.

### PR DESCRIPTION
**Pull request by Mryange**
## Proposed changes

The setting of _stop_emplace_flag is a heuristic operation. 
Previously, there was no batch processing on PipelineX (push one block, then pull one block), so it was easily misled by the first few small blocks. 
Therefore, batch processing is adopted here, and the first two values of STREAMING_HT_MIN_REDUCTION are set to 0. 
Only when enough data is obtained will _stop_emplace_flag be set to true.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Review by zhangstar333 — DISMISSED** [review]

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

File: be/src/pipeline/exec/distinct_streaming_aggregation_operator.cpp [hunk 1]
```diff
@@ -476,20 +477,23 @@ Status DistinctStreamingAggOperatorX::pull(RuntimeState* state, vectorized::Bloc
...
     return Status::OK();
 }
 
 bool DistinctStreamingAggOperatorX::need_more_input_data(RuntimeState* state) const {
     auto& local_state = get_local_state(state);
-    return local_state._aggregated_block->empty() && !local_state._child_eos &&
-           (_limit == -1 || local_state._output_distinct_rows < _limit);
+    const bool need_batch = local_state._stop_emplace_flag
+                                    ? local_state._aggregated_block->empty()
+                                    : local_state._aggregated_block->rows() < (state->batch_size());
```
Comment (HappenLee) [inline be/src/pipeline/exec/distinct_streaming_aggregation_operator.cpp:495]: check the batch_size() logic, maybe over the batch size

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Review by HappenLee — DISMISSED** [review]
LGTM

**Comment by Mryange** [comment]
run Code Formatter

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Review by Gabriel39 — APPROVED** [review]

**Pull request merged**